resources/sass/abstracts/_variables.scss

```diff
@@ -0,0 +1,13 @@
+$color_white:#fff;
```

Comment:
I don't see a proper consistent style guide implemented across the website. 

Things like, margins, paddings, fonts and other consistent css elements should have only one source of truth

Reply:
done